### DISCUSSION TITLE
How does the stun feature work?

### DISCUSSION THREAD

**Discussion by ManuelSvQ** [winstxnhdw/lc-hax#124] [Q&A]
The /stunclick and stun feature is making me confused, how does it work exactly? Is it just LMB if I have stunclick on?

**Comment by winstxnhdw** [comment]
Yeap.  `stun` just emulates an infinite flashbang.

**Reply by Dexteriority** [reply]
is it possible to make the stun also stop the movement of the enemy? I'm being chased by spider with esp off and I it still chasing me while dancing from the stunXDXD

**Reply by winstxnhdw** [reply]
Nope.

**Reply by Dexteriority** [reply]
Aight!
